client/src/services/AuthService/index.ts

```diff
@@ -16,7 +16,25 @@ const emailLogin = (payload: Object) => {
   });
 };
 
+const forgotPassword = (payload: Object) => {
```

Comment:
We need to send token also for secure routes, else it will not authenticate the server as we have added checks for authentication. 
Refer passwords service to know how we are using.